### PR TITLE
Feature/register page

### PR DESCRIPTION
**Pull request by miguelrnf**
Simulator Screen Shot - iPhone 13 - 2022-01-18 at 01 29 05

### CODE REVIEW

**Review by joaoachando — APPROVED** [review]

File: lib/Utils/Utils.dart [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  User,
+  ConfirmPassword
+
+}
+
+enum AssetType {
+  Facebook,
+  Google
+}
+
```
Comment (andretmarques) [inline lib/Utils/Utils.dart:18]: these are constants

File: lib/Utils/Utils.dart [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            ],
+          ),
+        ),
+      );
+    }
+
+    Widget buildTextButtonLabel(String text) {
+      return TextButton(child: Text(text,
+        style: GoogleFonts.roboto(
+            color: const Color.fromRGBO(88, 136, 204, 1),
```
Comment (andretmarques) [inline lib/Utils/Utils.dart:101]: constant

File: lib/login/LoginPage.dart [hunk 3]
```diff
@@ -35,19 +25,19 @@ class _LoginPageState extends State<LoginPage> {
...
+              utils.buildTextLabel("Please login to continue"),
+              utils.buildInput(InputType.Email),
+              utils.buildInput(InputType.Password),
+              Container(width: 350, child:utils.buildTextButtonLabel("Forgot Password?"), alignment: Alignment.centerRight),
               const Padding(padding: EdgeInsets.only(bottom: 50.0)),
-              _buildButton("LOGIN"),
-              Row( children: [_buildTextLabel("Don't have an account?"),
-                _buildTextButtonLabel('Create a new account')],
+              utils.buildButton("LOGIN"),
+              Row( children: [utils.buildTextLabel("Don't have an account?"),
```
Comment (andretmarques) [inline lib/login/LoginPage.dart:34]: lint

**Review by andretmarques — CHANGES_REQUESTED** [review]

File: lib/login/LoginPage.dart (hunk 3 above)
Comment (miguelrnf) [reply lib/login/LoginPage.dart]: Done [701bc68](https://github.com/andretmarques/UniHub/pull/6/commits/701bc68b4e1d87edfeebdda383e0fbddd8bf033a)

File: lib/Utils/Utils.dart (hunk 1 above)
Comment (miguelrnf) [reply lib/Utils/Utils.dart]: Done [701bc68](https://github.com/andretmarques/UniHub/pull/6/commits/701bc68b4e1d87edfeebdda383e0fbddd8bf033a)

File: lib/Utils/Utils.dart (hunk 2 above)
Comment (miguelrnf) [reply lib/Utils/Utils.dart]: Done [701bc68](https://github.com/andretmarques/UniHub/pull/6/commits/701bc68b4e1d87edfeebdda383e0fbddd8bf033a)

**Pull request merged**